Library/DiscUtils.Core/Plist.cs

```diff
@@ -37,7 +37,16 @@ internal static class Plist
 #if !NETCORE
             xmlDoc.XmlResolver = null;
 #endif
-            xmlDoc.Load(stream);
+
+            XmlReaderSettings settings = new XmlReaderSettings();
+#if !NET20
+            settings.DtdProcessing = DtdProcessing.Ignore;
```

Comment:
Put a note here, explaining that i MUST be set to Ignore on anything by net20. Then it should be good to go.